flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -657,6 +687,11 @@ public void selectionCleared() {
       rssDataSeries = stage.getRSSDataSeries();
     }
 
+    FlutterAllMemoryData.ThreadSafeData gcDataSeries = stage.getGcDataSeries();
```

Comment:
are these 3 variables needed. I don't see them being used but perhaps I am missing something.

Reply:
nm. perhaps move these variables down closer to where they are used.

Reply:
Yes when the DurationModel is created e.g.,

      new DurationDataModel<>(new RangedSeries<>(viewRange, new ResetStatsDataSeries(resetDataSeries)));

etc.